tests/subsys/fs/fs_api/src/test_fs_dir_file.c

```diff
@@ -142,16 +272,24 @@ void test_file_statvfs(void)
 
 	ret = fs_statvfs(NULL, &stat);
 	zassert_not_equal(ret, 0, "Get valume without path");
+	ret = fs_statvfs(TEST_FS_MNTP, NULL);
+	zassert_not_equal(ret, 0, "Get valume without stat structure");
```

Comment:
I think that the "value" has been misspelled in line 274 and the mistake has been carried here.
Why not just "pass NULL for stat structure pointer"?